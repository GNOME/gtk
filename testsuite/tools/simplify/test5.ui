<!-- placeholders are preserved -->
<interface>
  <object class="GtkWindow">
    <child>
      <object class="GtkBox">
        <child>
          <placeholder/>
        </child>
      </object>
    </child>
  </object>
</interface>
