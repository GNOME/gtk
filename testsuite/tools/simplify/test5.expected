<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow">
    <property name="child">
      <object class="GtkBox">
        <child>
          <placeholder/>
        </child>
      </object>
    </property>
  </object>
</interface>
